sdk/keyvault/azure-security-keyvault/azure/security/keyvault/secrets/README.md

```diff
@@ -0,0 +1,252 @@
+* [Create a Secret](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-security-keyvault#create-a-secret)
+* [Retrieve a Secret](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-security-keyvault#retrieve-a-secret)
+* [Update an existing Secret](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-security-keyvault#update-an-existing-secret)
+* [Delete a Secret](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-security-keyvault#delete-a-secret)
+* [List Secrets](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-security-keyvault#list-secrets)
+* [Async create a Secret](https://github.com/samvaity/azure-sdk-for-python/tree/secrets-track2-readme/sdk/keyvault/azure-security-keyvault/azure/security/keyvault/secrets#async-create-a-secret)
+* [Async list Secrets](https://github.com/samvaity/azure-sdk-for-python/tree/secrets-track2-readme/sdk/keyvault/azure-security-keyvault/azure/security/keyvault/secrets#async-list-secrets)
+
+### Create a Secret
+`set_secret` creates a Secret to be stored in the Azure Key Vault. If a secret with the same name already exists then a new version of the secret is created.
+```python
+    from azure.identity import AsyncClientSecretCredential
+    from azure.security.keyvault import SecretClient
+
+    credential = AsyncClientSecretCredential(client_id=client_id, secret=client_secret, tenant_id=tenant_id)
+    # Create a new secret client using a client secret credential
+    secret_client = SecretClient(vault_url=vault_url, credential=credential)
+
+    secret = secret_client.set_secret("secret-name", "secret-value", enabled=True)
+    
+    print(secret.id)
+    print(secret.value)
+    print(secret.version)
+    print(secret.enabled)
+```
+
+### Retrieve a Secret
+`get_secret` retrieves a secret previously stored in the Key Vault.
+```python
+    from azure.identity import AsyncClientSecretCredential
+    from azure.security.keyvault import SecretClient
+
+    credential = AsyncClientSecretCredential(client_id=client_id, secret=client_secret, tenant_id=tenant_id)
+    # Create a new secret client using a client secret credential
+    secret_client = SecretClient(vault_url=vault_url, credential=credential)
+
+    secret = secret_client.get_secret("secret-name")
+    
+    print(secret.id)
+    print(secret.value)
+```
+
+### Update an existing Secret
+`update_secret` updates a secret previously stored in the Key Vault.
+```python
+    from azure.identity import AsyncClientSecretCredential
+    from azure.security.keyvault import SecretClient
+
+    credential = AsyncClientSecretCredential(client_id=client_id, secret=client_secret, tenant_id=tenant_id)
+    # Create a new secret client using a client secret credential
+    secret_client = SecretClient(vault_url=vault_url, credential=credential)
+
+    content_type = "text/plain"
+    tags = {"foo": "updated tag"}
```

Comment:
Are the tags required here or is it just to show that they exist?

Reply:
just to show that they exist.

Reply:
And it would work if the secrets they have do not have any tags too, right?